plugin-taskbar/lxqttaskbutton.cpp

```diff
@@ -135,7 +136,12 @@ void LXQtTaskButton::updateIcon()
     }
     if (ico.isNull())
     {
-        ico = KWindowSystem::icon(mWindow);
+#if QT_VERSION >= 0x050600
+        int devicePixels = mIconSize * devicePixelRatioF();
```

Comment:
Couldn't we store (in `mIconSize`) the already multiplicated value? Or better store it only in some top object/widget...

... just thinking about (small) optimizations

Reply:
It seems more convenient to store the unmultiplied value in mIconSize, because then it's a direct comparison in changeEvent() to determine whether the value has changed.  And this way, we only do the multiplication if the value has, in fact, changed.

Reply:
Right... I was under impression that the `updateIcon()` is used more frequently... but now I see it's called only after a configuration/style/app change.